cpp/tests/io/parquet_common.cpp

```diff
@@ -479,7 +479,7 @@ std::enable_if_t<std::is_same_v<T, cudf::string_view>, cudf::test::strings_colum
 ascending()
 {
   auto elements = cudf::detail::make_counting_transform_iterator(0, [](auto i) {
-    std::array<char, 30> buf;
+    std::array<char, 30> buf{};
```

Comment:
To avoid compiler errors:

`/home/coder/cudf/cpp/tests/io/experimental/hybrid_scan_common.cpp:93:15: error: ‘buf’ may be used uninitialized [-Werror=maybe-uninitialized]`